src/js/components/Tabs/tabs.stories.js

```diff
@@ -361,6 +361,9 @@ const RichTabs = () => (
 
 const customTheme = deepMerge(grommet, {
   global: {
+    edgeSize: {
+      small: '6px',
```

Comment:
according to the documentation you added above
```
**global.edgeSize.small**

the possible sizes for margin, pad and gap Expects `string`.

Defaults to

``
6px
``
```
`global.edgeSize.small` defaults to `6px` so this value won't change its value.